changes-monitorsdoc (#1583)
SumoLogic/sumologic-documentation: docs/alerts/monitors/create-monitor.md

| Threshold type | How you want the value compared. Select either greater than, greater than or equal, less than or equal, or less than. |
| Threshold | The value against which the resolution will be evaluated. You can specify any valid numeric value. |

#### Alert and recovery window

This setting affects both the alert generation logic and the alert recovery logic.

![metrics alert datapoints.png](/img/monitors/minimum-datapoints.png)

> `Alert and recovery require a minimum of <Count> data points for `at all times` evaluation windows`

Reviewer: ```suggestion
`Alert and recovery require a minimum of <Count> data points for "at all times" evaluation windows`
```

This must be double quotes to match the UI (see screenshot below)
<img width="601" alt="Screenshot 2023-02-07 at 12 57 42 PM" src="https://user-images.githubusercontent.com/56411016/217364202-bbbcdc67-ab44-4bf4-a8c7-c9892e6a7b15.png">

| Parameter | Description |
|:--|:--|

> | Count | The minimum number of data points required within the 5-minute window to trigger an alert or recover from an alert. This means that if Sumo Logic receives fewer data points in a given window, no alert will be triggered (even if all those data points exceed the threshold).

Reviewer 2: The window might not always be 5 mins. `The minimum number of data points required within the configured window.

> | Count | The minimum number of data points required within the 5-minute window to trigger an alert or recover from an alert. This means that if Sumo Logic receives fewer data points in a given window, no alert will be triggered (even if all those data points exceed the threshold).

Reviewer: ```suggestion
| Count | The minimum number of data points required within the configured window to trigger an alert or recover from an alert. This means that if Sumo Logic receives fewer data points in a given window, no alert will be triggered (even if all those data points exceed the threshold).
```

For example, you want to be alerted when the CPU usage is over 60% `at all times` within a 5-minute window. If you set the count to 3, this means that you will only get an alert if you have at least 3 data points showing CPU usage above 60% within that 5-minute window. If you only have 2 data points, even if both of them show CPU usage above 60%, you won't get an alert.
:::note
This setting only works when you choose `at all times within` as the type of occurrence for the alert.  

> :::  |

Reviewer: ```suggestion
:::
```
